draft-ietf-quic-qpack.md

```diff
@@ -714,9 +714,9 @@ header field name matches the header field name of an entry stored in the static
 table or the dynamic table.
 
 If the entry is in the static table, or in the dynamic table with an absolute
-index less than or equal to Base Index, this representation starts with the '00'
+index less than or equal to Base Index, this representation starts with the '01'
 two-bit pattern.  If the entry is in the dynamic table with an absolute index
-greater than Base Index, the representation starts with the '0101' four-bit
+greater than Base Index, the representation starts with the '0000' four-bit
 pattern.
```

Comment:
This change is fine, but didn't we separate out the Post-Base things to their own headers?  Did we miss one?